test/bpf/selective_drop.c

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+//
+
+#include "bpf_endian.h"
+#include "bpf_helpers.h"
+
+// Map configured by user mode to drop packets from a specific interface.
+struct
+{
+    __uint(type, BPF_MAP_TYPE_ARRAY);
+    __type(key, uint32_t);
+    __type(value, uint32_t);
+    __uint(max_entries, 1);
+} interface_map SEC(".maps");
+
+// Map to track the number of dropped packets.
+struct
+{
+    __uint(type, BPF_MAP_TYPE_ARRAY);
+    __type(key, uint32_t);
+    __type(value, uint64_t);
+    __uint(max_entries, 1);
+} dropped_packet_map SEC(".maps");
+
+SEC("xdp/selective_drop")
+int
+drop(xdp_md_t *ctx)
```

Comment:
for consistency, our programs so far have used the same function name as driver name and section name, so selective_drop is preferred here.

Reply:
Fixed.